main.tex

```diff
@@ -50,99 +61,204 @@
+We define \emph{CoinJoin inefficiency} as the fraction of non-mixed change outputs in a CoinJoin transaction. In~\cref{fig:cjinefficiency}, we observe the large amount of non-mixed outputs of essentially all Bitcoin privacy-enhancing tools. Samourai has the lowest CoinJoin inefficency, since it always applies $5$ equal amount of inputs and outputs. We note a sharp decline of CoinJoin inefficiency in Wasabi CoinJoins around January 2019 due to the introduction of multiples of the base denomination.
 
 \begin{figure}[h!]
     \centering
     \includegraphics[scale=0.4]{Figures/CJInefficiency.pdf}
-    \caption{CoinJoin inefficiency of various privacy-focused Bitcoin wallets.}
+    \caption{CoinJoin inefficiency of various privacy-focused Bitcoin wallets. We define CoinJoin inefficiency to be the fraction of non-mixed change outputs and the total number of outputs in a CoinJoin transaction. A higher value indicates more block space required to mix a given volume of Bitcoin. All CoinJoin implementations tools produce a significant amount of ``toxic change" due to the requirement of mixed outputs being uniform, either directly from CoinJoin transactions or in Samourai's case from antecessor transactions that prepare outputs for mixing.}
     \label{fig:cjinefficiency}
 \end{figure}
 
 \subsubsection{Minimum denomination}
 
-In order to participate, a user's combined input amount must be greater or equal to the base denomination.\footnote{The observed base denominations in Wasabi's CoinJoins are usually slightly higher than the announced, agreed upon base denomination. Thus participants sometimes get back slightly more value in the CoinJoins than they put in.} We observe, that considerable portion of CoinJoin inputs are less than this minimum denomination, see \cref{fig:minimumdenomination}.
+In order to participate in a CoinJoin, a user's combined input amount must be greater or equal to the base denomination.\footnote{The observed base denominations in Wasabi's CoinJoins are usually slightly higher than the announced, agreed upon base denomination. Thus, participants sometimes get back slightly more value in the CoinJoins than they put in.} We observe, that considerable portion of CoinJoin inputs are less than this minimum denomination, see \cref{fig:minimumdenomination}.
 
-\begin{figure}[h!]
+\begin{figure}%[ht!]
+\centering
+\begin{minipage}{.45\textwidth}
     \centering
-    \includegraphics[scale=0.4]{Figures/SmallValueInputsWasabi.pdf}
+    \includegraphics[width=\linewidth]{Figures/SmallValueInputsWasabi.pdf}
     \caption{Fraction of inputs with value smaller than the minimum denomination in Wasabi CoinJoin transactions.}
     \label{fig:minimumdenomination}
+\end{minipage}
+\hspace{6pt}
+\begin{minipage}{.45\textwidth}
+    \centering
+    \includegraphics[width=0.95\linewidth]{Figures/freshBitcoins.pdf}
+    \caption{Volume of unmixed coins entering to various CoinJoin schemes. Wasabi is the most popular Chaumian CoinJoin implementation.}
```

Comment:
Here we use `unmixed` while in other places we call them `non-mixed`. Is this okay?

Reply:
to me non-mixed implies something more intrinsic than unmixed, the latter kind of suggests that coins can start unmixed and then become mixed which is technically not true, so I think that's how we ended up with that, i think consistency is nice, but we should probably ask a native speaker if either of them sounds weird and if the connotation i'm getting is just in my head... @DanGould, do you have any thoughts?

Reply:
I try to wholly avoid the word "mix" because it's so ambiguous to many on-chain practices including some unscrupulous custodial services. Perhaps un-joined. Or standard use. or common-input-heuristic-vulnerable. or "cluster vulnerable".

"Non-fungible" is making it's way into the vernacular and also describes the un-mixed property more concretely as it relates to what the mixing actually does. I like that the best.

p.s. not sure how I managed to lose this mention for so long